### PR TITLE
Updated with instructions

### PR DESCRIPTION
**Pull request by Soraph**

### CODE REVIEW

**Comment by adjohnson916** [comment]
@Soraph, thanks, this should help people that aren't using Bower. However, I would like to keep my Bower instructions there as well in a first "Bower" section, then scope your instructions under a "Manual" section. Is that something you can correct in your patch then rebase? Otherwise I can do a manual merge.

File: README.md [hunk 1]
```diff
@@ -16,11 +16,30 @@ Either [download manually](https://github.com/adjohnson916/magnificent.js/releas
 $ bower install --save jquery-magnificent
 ```
 
-In your page, include:
-* All files specified in the `main` property of the [bower.json](bower.json) file.
-* All dependencies, as specified in the `dependencies` property of the [bower.json](bower.json) file.
+For the basic setup include in your page the following depencies:
```
Comment (adjohnson916) [inline README.md]: Typo here: "depencies" should be "dependencies".

File: README.md [hunk 2]
```diff
@@ -16,11 +16,30 @@ Either [download manually](https://github.com/adjohnson916/magnificent.js/releas
...
+<script src="https://ajax.googleapis.com/ajax/libs/jquery/2.1.4/jquery.min.js"></script>
+<script type="text/javascript" src="path/to/js/jquery-bridget.js"></script>
+<script type="text/javascript" src="path/to/js/mag.js"></script>
+<script type="text/javascript" src="path/to/js/mag-jquery.js"></script>
+```
+To include jquery-bridget please refer to [jquery-bridget github project][jquery-bridget]
+
+To use the additional functionality described in the [demo][mag-demo], include also:
+
+[https://github.com/jquery/jquery-mousewheel][jquery-mousewheel]
```
Comment (adjohnson916) [inline README.md]: Please add "For scroll zoom:".

File: README.md [hunk 3]
```diff
@@ -16,11 +16,30 @@ Either [download manually](https://github.com/adjohnson916/magnificent.js/releas
...
+
+[https://github.com/threedubmedia/jquery.threedubmedia][jquery-threedubmedia]
```
Comment (adjohnson916) [inline README.md]: Please add "For drag events:".

**Comment by Soraph** [comment]
I've updated the description, I never used bower so I don't know how it structure the folders when you download something, please manually add the necessary missing information where needed, my patch want to be more of a guideline than anything else

**Comment by adjohnson916** [comment]
@Soraph, thanks. I'll add something like your change guidelines then.

Anyway I'm actually moving the README to be generated by `verbose/verb` in the `verb` branch - so I'll make the changes in the `.verb.md` file there.

**Pull request closed without merging**